### ISSUE TITLE
Can't reproduce the results of standalone_simple_example.ipynb

### ISSUE THREAD

**Issue by hangyao** [ragulpr/wtte-rnn#27]
Hi, 

I'm trying to reproduce the results in examples/keras/standalone_simple_example.ipynb, but the genrated data are different than what is shown in this notebook.

Below are the data plotted in my local notebook. I use Python 3.5 and Windows 7 OS. With these data sets, the neural network doesn't learn at all.

Another notebook simple_example.ipynb is working fine.

![1](https://user-images.githubusercontent.com/19369631/28690901-831e5400-72e0-11e7-88b1-dac9aebfde21.png)
![2](https://user-images.githubusercontent.com/19369631/28690903-831f72a4-72e0-11e7-832d-8636b6ba55a9.png)
![3](https://user-images.githubusercontent.com/19369631/28690902-831f0562-72e0-11e7-9ca5-820e4668c437.png)
![4](https://user-images.githubusercontent.com/19369631/28690904-8328ed84-72e0-11e7-9589-2e623459fca0.png)
![5](https://user-images.githubusercontent.com/19369631/28690905-832f49fe-72e0-11e7-85ef-4895d67a003f.png)

**Comment by tom-christie** [comment]
@hangyao I'm working through this notebook too.  I think this line:

    x_train = np.tile(x_train.T,n_repeats).T

should be:

    x_train = np.repeat(x_train,n_repeats, axis=0)

There's a (unintentional?) time shift that makes the vertical lines in x_test become staggered in x_train.  Changing the line in question fixes that problem and makes x_train look a lot more like x_test.

There's still some other problem I'm running into, as the learned alpha value is very high and constant (near 40).  If I figure that one out I'll report back.

**Comment by ragulpr** [comment]
Thanks for reporting, I haven't run `standalone_simple_example.ipynb` for a while, it must be numpy-api changes or something of the sort. I'd copy paste use the data-generating code from `simple_example.ipynb` instead, that works.
